semm10-sp2

MaSemerad opened the pull request:
Link k aplikaci: https://eso.vse.cz/~semm10/sp2/

MaSemerad committed:
semm10-sp2

Link k aplikaci: https://eso.vse.cz/~semm10/sp2/

nvbach91 commented:
- ovladaci prkvy formulare maji ve formulari s udalosti submit
- pro volani na REST API pouzivejte funkci `fetch`, nikoliv `XMLHttpRequest`
- highscore nema byt jedna hodnota, ale seznam hodnot, ktery se bude zobrazovat jako tabulka

nvbach91 commented:
- nesmi se zasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, zmena obsahu elementu, ...)
- ajax loading, data loading indikace - animace

MaSemerad committed:
semm10-sp2 v2

Změny: 
přidání form s eventem submit
DOM se provolává jen 1x při tvoření leaderboard tabulky
volání REST API skrze fetch
přidán spinner, který se točí při provolávání REST API dat
highscore je nově tabulka s TOP 10 scory

MaSemerad commented:
Přikládám komentář se změny, které jsem provedl v pull reuestu
Změny: 
přidání form s eventem submit
DOM se provolává jen 1x při tvoření leaderboard tabulky
volání REST API skrze fetch
přidán spinner, který se točí při provolávání REST API dat
highscore je nově tabulka s TOP 10 scory
